contracts/PLCR.sol

```diff
@@ -120,11 +120,11 @@ contract PLCR is AragonApp, IVoting {
         require(getTimestamp() <= vote.commitEndDate);
 
         // check lock and get amount
-        uint256 stake = checkLock(msg.sender, _lockId, vote.revealEndDate);
+        uint256 stake = checkLock(msg.sender, _lockId, vote.commitEndDate, vote.revealEndDate);
```

Comment:
This is not enough: the user could unlock the remaining tokens (`stake - slashStake`) to another account before the reveal period starts (i.e., the commit period ends) and vote again.